extensions/cornerstone/src/commandsModule.ts

```diff
@@ -491,33 +493,55 @@ function commandsModule({ servicesManager, commandsManager }) {
         }
       }
     },
-    firstImage: () => {
-      // Get current active viewport (return if none active)
-      const enabledElement = _getActiveViewportEnabledElement();
-      if (!enabledElement) {
-        return;
-      }
-      const { viewport } = enabledElement;
 
-      // Check viewport is supported
-      if (
-        viewport! instanceof StackViewport &&
-        viewport! instanceof VolumeViewport
-      ) {
-        throw new Error('Unsupported viewport type');
+    /**
+     * This will clear the presentation properties which are being
+     * updated in the given viewport and return an appropriate
+     * viewportOptions to use to update those properties.
+     * Currently only supports imageIndex, but the intent is to support
+     * other parameters to allow initial display with the already updated
+     * viewport data.
+     */
+    clearPresentation: props => {
+      const { viewport, imageIndex } = props;
+      let { positionPresentationStore } = stateSyncService.getState();
+
+      const positionId =
+        viewport.viewportOptions?.presentationIds?.positionPresentationId;
+      if (positionId && positionPresentationStore[positionId]) {
+        const updatePosition = {
+          ...positionPresentationStore[positionId],
+          initialImageIndex: imageIndex,
+        };
+        positionPresentationStore = {
+          ...positionPresentationStore,
+          [positionId]: updatePosition,
+        };
```

Comment:
I guess this can be merged 

```js
positionPresentationStore[positionId] = {
  ...positionPresentationStore[positionId],
  initialImageIndex: imageIndex,
};
```